src/averageAges.js

```diff
@@ -15,6 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  let man = [];
+
+  if (!century) {
+    man = people.filter(person => person.sex === 'm');
+  } else {
+    man = people.filter(person =>
+      person.sex === 'm' && Math.ceil(person.died / 100) === century);
+  }
```

Comment:
I would always assign all males at the begining: 
```
let men = people.filter(person => person.sex === 'm');
if(some condition) { 
///filter men from century
} else  {
//return calculateSum(...)
}
```

Your solution is correct, but you could extract filtering into one place.